report/src/main/webapp/app/layout/main/url/reports/source.diff.html

```diff
@@ -0,0 +1,82 @@
+<!--
```

Comment:
Could you please rename this file to `source-diff.html` or `source_diff.html` so that we dont introduce another convention into reports templates names?

I mean: other files use `-` or `_` as word separtor in their names.

Reply:
done